src/ios/formatters/__tests__/spacingFormatter.test.ts

```diff
@@ -0,0 +1,49 @@
+import createDictionary from 'style-dictionary/lib/utils/createDictionary'
+import createFormatArgs from 'style-dictionary/lib/utils/createFormatArgs'
+
+import { TEMPLATES } from '../../constants'
+import { FormatterConfig } from '../../../common/types'
+import { spacingFormatter } from '../spacingFormatter'
+import { HEADER } from './constants'
+
+describe('spacingFormatter', () => {
+  it('can handle dictionary with spacing tokens', () => {
+    const { spacingsTemplate } = TEMPLATES('', 'DesignSystem')
+
+    const spacingConfig: FormatterConfig = {
+      template: spacingsTemplate.source,
+      header: HEADER,
+      packageName: 'nl.elements.skeleton',
+    }
+
+    const properties = {
```

Comment:
These could probably be shared between the tests. We could create a file with all the properties together.

Reply:
Done